docs/metrics.md

```diff
@@ -0,0 +1,171 @@
+# METRICS
+
+## Data Analysis
+The collected data will primarily be used to answer the following questions.
+Images are used for visualization and are not composed of actual data.
+
+### Do users install and run this?
+
+What is the overall engagement of the Containers experiment?
+**This is the standard Daily Active User (DAU) and Monthly Active User (MAU) analysis.**
+
+This captures data from the users who have the add-on installed, regardless of
+whether they are actively interacting with it.
+
+![](images/kpi-1.png)
+
+### Immediate Questions
+
+* Do people use the containers feature & how do people create new container tabs?
+  * Click to create new container tab
+    * \+ `entry-point` value: "tab-bar" or "pop-up"
+* Do people who use the containers feature continue to use it?
+  * Retention: opening a second container tab (second tab in the same container, or a tab in a second container?)
```

Comment:
We will have a histogram over time of all the events, including opening of containers. We honestly don't know - users might use them all the time, or they may only use 1 container tab once every few weeks. E.g., just when they do their online banking. Here we will simply measure feature usage, and we may have to learn user *intent* or *experience* with surveys.